### ISSUE TITLE
Adding jet and electron ID related variables to 2017 analysis

### ISSUE THREAD

**Issue by eyigitba** [UFLX2MuMu/Ntupliser#94]
The missing items are listed here: https://docs.google.com/spreadsheets/d/1aGZC4l2LytWQBGmxWcy0ejRYXKKzl50mCupPfTOj0sA/edit#gid=0

We want to save jetID in ntuples, change electron ID cut to POG MVA wp90 and apply cut-based preselection cuts to align electron quality to the baseline tZq selection. 

Relevant information can be found here:

- https://twiki.cern.ch/twiki/bin/view/CMS/MultivariateElectronIdentificationRun2
- https://twiki.cern.ch/twiki/bin/viewauth/CMS/CutBasedElectronIdentificationRun2
- http://cms.cern.ch/iCMS/jsp/db_notes/noteInfo.jsp?cmsnoteid=CMS%20AN-2018/100

**Comment by eyigitba** [comment]
I added the variables. The commit is here: https://github.com/UFLX2MuMu/Ntupliser/commit/7059da62e909167f0aaab9bbbcb9ae1ae33f8849.

There are a couple of things I wanted to ask. Most of them are preference.

Jets:

- We are never using "loose" ID. So why do we want to record "isTightID" for jets? All of them are "tight" anyway. Do we want to loosen the cuts and then record the variable so we can cut on this later in the analyzer?


Electrons:

1. Which electron eta is the correct one? SuperCluster eta or actual ele->eta()?
2. I created a new "ID" and named it "tZq". We can name it "custom" or something else if it sounds better. We need to change our cut based ID cut to "tZq" for synchronization with ttH. Do we want to keep these working points still? Do we want to create a "istZqID" flag to check on analyzer side? Once again this will be a trivial flag since we're only accepting "tZq" ID electrons.
3. I made all the custom cuts modifiable in the config files. This adds ~10 lines to the config files which will probably remain the same (at least for now). We can move them to hard-coded if we want.

Let me know what you think.

**Comment by zuoxunwu** [comment]
For electrons, could you save such flags:
POG MVA loose
POG MVA wp90
tZq ID
POG cut-based loose

Thanks,
Xunwu

**Comment by eyigitba** [comment]
I applied the changes we discussed on Monday. Also added POG MVA loose ID flag Xunwu requested. The commits are below. I am also submitting a PR for this.

https://github.com/UFLX2MuMu/Ntupliser/commit/8296af03fa0d02d5d19d34421edddf08e9ced283
https://github.com/UFLX2MuMu/Ntupliser/commit/b29d2e881634ffb9db6a130b3afe0fcea803ea74 
https://github.com/UFLX2MuMu/Ntupliser/commit/1539bf5a5430967b3cba8cda5e5f44642990736f

I changed the config files to keep all electrons without applying an ID cut. The file size increase in ggH MC was ~1% and in data was ~10%. If you prefer we don't do this we can revert that change.
